port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -230,6 +251,10 @@ async def _parse_items(
                         examples_to_send.append(result.raw_data)
                 else:
                     failed_entities.append(parsed_entity)
+        if len(entity_misconfigurations) > 0:
+            logger.info(
```

Comment:
wouldn't it be too spammy? for each entity?

Reply:
I think it should be in `debug` mode and we should log it in `info` as an aggregated mode for a batch of entities

Reply:
we've discussed this 🙏